hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/config/types/KeyValuePair.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.spi.config.types;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import edu.umd.cs.findbugs.annotations.Nullable;
+import java.util.Objects;
+
+/**
+ * A simple key-value pair. This record is used to create {@link java.util.Map} like structures for config data
+ * properties. See {@link com.hedera.node.app.spi.config.validation.EmulatesMap} for more details.
+ */
+public record KeyValuePair(@NonNull String key, @Nullable String value) {
```

Comment:
If `value` can be `null`, what does it mean? What is the difference between `null` and an empty `String`? Maybe you document that somewhere.

Reply:
you are right. Null is not possible